catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesLuceneIndexer.java

```diff
@@ -229,10 +228,10 @@ private void addDocument(final IndexWriter indexWriter, final GeoEntry geoEntry,
                 geoEntry.getName(),
                 Field.Store.YES));
 
-        document.add(new DoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
+        document.add(new LegacyDoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
```

Comment:
I looked at DoublePoint, but its ctor does take the Field.Store.YES argument. I didn't know how that would change the behavior.